app/assets/javascripts/selectors/done.js

```diff
@@ -1,13 +1,12 @@
 import moment from "moment";
 
-export const mountPastIterations = (pastIterations) => {
-  const sprints = pastIterations.map((sprint, index) => ({
-    ...sprint,
-    number: index + 1,
-    startDate: moment(sprint.startDate).format("ddd MMM Do Y"),
-    endDate: moment(sprint.endDate).format("ddd MMM Do Y"),
-    stories: [],
-  }));
-
-  return sprints;
-};
+export const mountPastIterations = (pastIterations) =>
+  pastIterations.map(sprint =>
+    ({
```

Comment:
Can we put it in the same line as the arrow? It follows the standard we use for the rest of the app.